scripts/docker/debian/Dockerfile

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# Dockerfile that creates a container suitable to build dotnet-cli
+FROM debian:jessie
+
+# Misc Dependencies for build
+RUN apt-get update && apt-get -qqy install curl unzip gettext sudo
+
+# This could become a "microsoft/coreclr" image, since it just installs the dependencies for CoreCLR (and stdlib)
+RUN echo "deb http://llvm.org/apt/jessie/ llvm-toolchain-jessie-3.6 main"  | tee /etc/apt/sources.list.d/llvm.list && \
+    curl http://llvm.org/apt/llvm-snapshot.gpg.key | apt-key add - && \
+    apt-get update && apt-get -qqy install\
+	libc6 \
+	libedit2 \
+	libffi6 \
+	libgcc1 \
+	libicu52 \
+	liblldb-3.6 \
```

Comment:
@Sridhar-MS 
- Yes with these changes debian tarballs are produced by the builds.  As I eluded to in my summary comments, I have a VSO build definition hooked up to my fork that is uploading these to azure.  This build definition will need to be modified when this PR is merged.
- Jenkins - I wasn't sure the Jenkins integration was necessary for RC2 as I listed in my summary.  Regardless can debian builds be added to Jenkins prior to merging this PR?